sda-commons-client-jersey-example/src/main/java/org/sdase/commons/client/jersey/clients/apia/Car.java

```diff
@@ -1,24 +1,3 @@
 package org.sdase.commons.client.jersey.clients.apia;
 
-public class Car {
-  private String sign;
-  private String color;
-
-  public String getSign() {
-    return sign;
-  }
-
-  public Car setSign(String sign) {
-    this.sign = sign;
-    return this;
-  }
-
-  public String getColor() {
-    return color;
-  }
-
-  public Car setColor(String color) {
-    this.color = color;
-    return this;
-  }
-}
+public record Car(String sign, String color) {}
```

Comment:
I guess, there is more version related code smell like this? Any idea how we want to migrate the code?

Reply:
It's possible to use InteeliJ's feature "Analyze > Inspect code" and look for "Java > Java language level migration aids". Let me check what I can find there.